cookbooks/tomography_based_plate_motions/2D_slice_with_faults_slabs_and_topo.prm

```diff
@@ -0,0 +1,168 @@
+    set Inner radius  = 3481000
+    set Outer radius  = 6371000
+  end
+
+  subsection Initial topography model
+    set Model name = ascii data
+
+    subsection Ascii data model
+      set Data directory       = $ASPECT_SOURCE_DIR/cookbooks/tomography_based_plate_motions/input_data/
+      set Data file name       = input_topography_smooth_gauss.txt
+    end
+  end
+
+#  subsection Initial topography model
+#    set Model name = function
+#
+#    subsection Function
+#      set Variable names      = r, phi
+#      set Function constants  = h = 8e3, n_phi = 1
+#      set Coordinate system   = spherical
+#      set Function expression = h*sin(n_phi*phi)
+#    end
+#  end
+end
+
+# We use adaptive refinement to better resolve the plate boundaries and
+# the near-surface heterogeneity
+subsection Mesh refinement
+  set Initial adaptive refinement = 4
+  set Initial global refinement = 4
+  set Strategy = minimum refinement function, composition threshold
+  set Skip solvers on initial refinement = true
+
+  subsection Minimum refinement function
+    set Variable names = depth, y
+    set Function expression = if(depth > 350000, 5, 7 )
+  end
+
+  subsection Composition threshold
+    set Compositional field thresholds = 1e6, 1e6, 1e6, 1e6, 1e6, 0.2
+  end
+end
+
+subsection Compositional fields
+  set Number of fields = 6
+  set Names of fields = grain_size, Vp, Vs, vs_anomaly, faults, slabs
+  set Compositional field methods = prescribed field, static, static, static, static, prescribed field with diffusion
+end
+
+
+# We use world builder to define the complex geometery of plate boundaries ("faults")
+# and cratons ("continents") in our model.
+subsection Initial composition model
+  set List of model names = ascii data, world builder, slab model
+
+  subsection Ascii data model
+    set Data directory = $ASPECT_SOURCE_DIR/cookbooks/tomography_based_plate_motions/input_data/
+    set Data file name = LLNL_model_cropped_cratons_faults.txt.gz
+    set Slice dataset in 2D plane = true
+  end
+
+  subsection World builder
+    set List of relevant compositions = faults
+  end
+
+  subsection Slab model
+    set Data directory = $ASPECT_SOURCE_DIR/cookbooks/tomography_based_plate_motions/input_data/
+    set Data file name = slab2_depth_thickness_2D.txt
+  end
+end
+
+subsection Boundary velocity model
+  set Tangential velocity boundary indicators = bottom, top
+end
+
+
+# The material model uses diffusion/dislocation creep with prefactors, activation
+# energies and volumes for each major mantle phase chosen to facilitate combined
+# diffusion/dislocation creep in the upper mantle and transition zone, and diffusion
+# creep as the dominant deformation mechanism in the lower mantle.
+subsection Material model
+  set Model name         = tomography based plate motions
+  set Material averaging = harmonic average only viscosity
+
+  subsection Tomography based plate motions model
+    set Use cratons                                 = false
```

Comment:
Can you provide some comments on what this parameter does and why it is set to false? Other places in the PRM file say the model includes Cratons, so some additional explanation would be helpful here.

Reply:
Thanks for the catch! It was mostly out of convenience that I ignored cratons because the way I had the models setup, adding another compositional field (i.e, slabs) would also require me to change the accompanying ascii data file to include `slabs` compositional field as well. I have kept the cratons in for consistency.

Reply:
Yes, please just add a comment summarizing what this parameter does, why it is set to fault, and how this fits in with the rest of the model setup and documentation.